src/components/05-alerts/alerts.njk

```diff
@@ -1,42 +1,8 @@
-
-  <div class="usa-alert usa-alert-success">
-    <div class="usa-alert-body">
-      <h3 class="usa-alert-heading">Success Status</h3>
-      <p class="usa-alert-text">Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.</p>
-    </div>
-  </div>
-
-  <div class="usa-alert usa-alert-warning">
-    <div class="usa-alert-body">
-      <h3 class="usa-alert-heading">Warning Status</h3>
-      <p class="usa-alert-text">Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.</p>
-    </div>
-  </div>
-
-  <div class="usa-alert usa-alert-error" role="alert">
-    <div class="usa-alert-body">
-      <h3 class="usa-alert-heading">Error Status</h3>
-      <p class="usa-alert-text">Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.</p>
-    </div>
-  </div>
-
-  <div class="usa-alert usa-alert-info">
-    <div class="usa-alert-body">
-      <h3 class="usa-alert-heading">Information Status</h3>
-      <p class="usa-alert-text">Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.</p>
-    </div>
-  </div>
-
-  <div class="usa-alert usa-alert-info">
-    <div class="usa-alert-body">
-      <h3 class="usa-alert-heading">Information Status</h3>
-      <p class="usa-alert-text">Multi line. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui atione voluptatem sequi nesciunt. Neque porro quisquam est, qui doloremipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
-    </div>
-  </div>
-
-  <div class="usa-alert usa-alert-info usa-alert-paragraph">
-    <div class="usa-alert-body">
-      <h3 class="usa-alert-heading">Information Status - Paragraph Width</h3>
-      <p class="usa-alert-text">Multi line. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui atione voluptatem sequi nesciunt. Neque porro quisquam est, qui doloremipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
-    </div>
-  </div>
+<div class="usa-alert {{ alert.classes }}" {% if alert.role %}role="{{alert.role}}"{% endif %}>
+  <div class="usa-alert-body">
+    {% if alert.title %}
+    <h3 class="usa-alert-heading">{{ alert.title }}</h3>
+    {% endif %}
+    <p class="usa-alert-text">{{ alert.content }}</p>
```

Comment:
Since these fractal components are becoming more likely to be included in the docs (see e.g. https://github.com/18F/web-design-standards-docs/pull/357), I'm curious if we should start using `{%-` and `-%}` to [remove excess blank lines](https://mozilla.github.io/nunjucks/templating.html#whitespace-control) from the resulting HTML code?  Using it on the above code, for instance, would get rid of the blank lines in the HTML rendering:

```html
<div class="usa-alert usa-alert-info">
    <div class="usa-alert-body">

        <h3 class="usa-alert-heading">Information Status</h3>

        <p class="usa-alert-text">Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.</p>
    </div>
</div>
```

It also makes the code snippet nicer to read on the fractal site too, of course, but it's really up to you.